client/blocks/login/two-factor-authentication/waiting-notification-approval.jsx

```diff
@@ -74,47 +58,29 @@ class WaitingTwoFactorNotificationApproval extends Component {
 	};
 
 	render() {
-		const {
-			isSmsAuthSupported,
-			translate,
-		} = this.props;
+		const { translate, twoFactorAuthType } = this.props;
 
 		return (
 			<form>
 				<Card className="two-factor-authentication__push-notification-screen is-compact">
 					<p>
-						{ translate( 'We just sent a push notification to your WordPress mobile app. ' +
-							'Swipe or tap to open and verify your login.' ) }
+						{ translate( 'We sent a push notification to your {{strong}}WordPress mobile app{{/strong}}. ' +
+							'Once you get it and swipe or tap to confirm, this page will update.', {
+								components: {
+									strong: <strong />
+								}
+							} ) }
 					</p>
 					<div>
 						<img className="two-factor-authentication__auth-code-preview"
 							src="/calypso/images/login/pushauth.svg" />
 					</div>
 				</Card>
-				<Card className="two-factor-authentication__form-action is-compact">
-					<p>
-						{ translate( 'Or, type in the verification code generated by your Authenticator app' +
-							' or sent via text.' ) }
-					</p>
-					<p>
-						<Button onClick={ this.verifyWithCodeInstead }>
-							{ translate( 'Verify with code instead' ) }
-						</Button>
-					</p>
-				</Card>
-				<p>
-					<ExternalLink
-						icon={ true }
-						target="_blank"
-						href="http://en.support.wordpress.com/security/two-step-authentication/">
-						{ translate( 'Help' ) }
-					</ExternalLink>
-				</p>
-				{ isSmsAuthSupported && (
-					<p>
-						<a href="#" onClick={ this.sendSmsCode }>{ translate( 'A recovery code via text' ) }</a>
-					</p>
-				) }
+
+				<TwoFactorActions
+					errorNotice={ this.props.errorNotice }
+					successNotice={ this.props.successNotice }
+					twoFactorAuthType={ twoFactorAuthType } />
```

Comment:
Mentioned elsewhere, but just to make it clearer: this can be `twoFactorAuthType="push"` since it will never be different for this component.